kroxylicious-test-tools/pom.xml

```diff
@@ -22,79 +22,96 @@
     <name>Testing tools</name>
 
     <dependencies>
+
+        <!-- project dependencies - runtime and compile -->
         <dependency>
-            <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka-clients</artifactId>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious</artifactId>
         </dependency>
+
+        <!-- project dependencies - test -->
         <dependency>
             <groupId>io.kroxylicious.testing</groupId>
-            <artifactId>testing-junit5-extension</artifactId>
+            <artifactId>testing-api</artifactId>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>io.kroxylicious.testing</groupId>
-            <artifactId>testing-api</artifactId>
+            <artifactId>testing-junit5-extension</artifactId>
             <scope>compile</scope>
         </dependency>
+
+        <!-- third party dependencies - runtime and compile -->
         <dependency>
-            <groupId>io.kroxylicious</groupId>
-            <artifactId>kroxylicious</artifactId>
-            <version>0.3.0-SNAPSHOT</version>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-common</artifactId>
+            <scope>compile</scope>
```

Comment:
The PR only adds the `<scope>compile</scope>` in this pom, is there a reason for doing here specifically?

Reply:
I've backed out this part of the change.